spec/system/case_court_reports/index_spec.rb

```diff
@@ -8,12 +8,13 @@
   let(:at_least_transition_age) { volunteer.casa_cases.find(&:in_transition_age?) }
   let(:modal_selector) { '[data-bs-target="#generate-docx-report-modal"]' }
 
+  let(:date) { Date.current }
+  let(:formatted_date) { date.strftime("%B %d, %Y") } # January 01, 2021
+
   before do
-    travel_to Date.new(2021, 1, 1)
     sign_in volunteer
     visit case_court_reports_path
   end
-  after { travel_back }
```

Comment:
This caused a flaky spec. Even thought there was an explicit `travel_back` here, the spec would fail because the user would get logged out... I'm guessing there's a race condition and `before { sign_in user }` happens in 2021, but we get back to present day before request/page load, and we redirect to root?

It's pretty common to use `travel_to` to test dates in ways that are more readable like this. In general, not just in this app. So may be something I'm missing here.

May be worth making a helper for asserting dates in specific formats on the page to avoid this elsewhere. Maybe even using i18n format yaml?